modules/foreman_operating_system.py

```diff
@@ -165,14 +165,12 @@
         PartitionTable,
         ProvisioningTemplate,
     )
+except ImportError:
+    pass
 
-    has_import_error = False
-except ImportError as e:
-    has_import_error = True
-    import_error_msg = str(e)
-
-from ansible.module_utils.basic import AnsibleModule
-
+from ansible.module_utils.foreman_helper import (
```

Comment:
Any particular reason this line is split where in other files it isn't?

Reply:
artifact from the time I've imported multiple things from the helper